internal/operands/data-sources/reconcile.go

```diff
@@ -217,66 +205,254 @@ func reconcileEditRole(request *common.Request) (common.ReconcileResult, error)
 		Reconcile()
 }
 
-type dataSourcesAndCrons struct {
-	dataSourceInfos []dataSourceInfo
-	dataImportCrons []cdiv1beta1.DataImportCron
+func (d *dataSources) getDataSourcesAndCrons(request *common.Request) (dataSourcesAndCrons, error) {
+	isMultiarch := ptr.Deref(request.Instance.Spec.EnableMultipleArchitectures, false)
+
+	var cronByDataSource map[client.ObjectKey]cdiv1beta1.DataImportCron
+	if isMultiarch {
+		var err error
+		cronByDataSource, err = getCronsByDataSourceMultiArch(d.sourceCollection, &request.Instance.Spec)
```

Comment:
Swap order of func arguments on `getCronsByDataSourceMultiArch` to make it more similar to the regular `getCronsByDataSource`?